staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go

```diff
@@ -118,6 +124,10 @@ func sysctlsV1Dot29(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) Che
 	return sysctls(podMetadata, podSpec, sysctlsAllowedV1Dot29)
 }
 
+func sysctlsV1Dot30(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
```

Comment:
I'm not 100% sure this is correct/needed, I'd like a more experienced reviewer to confirm it.

Reply:
Yeah, let's wait for an experienced member. Even I also have doubt for the same.